src/docstring/get_template.ts

```diff
@@ -8,6 +8,8 @@ export function getTemplate(docstringFormat: string): string {
             return getTemplateFile("sphinx.mustache");
         case "numpy":
             return getTemplateFile("numpy.mustache");
+	case "oneline-rst":
```

Comment:
The indentation looks weird here. Can you use spaces?

Reply:
yes! sorry about that